src/app/fsat/fsat-fluid/fsat-fluid.component.html

```diff
@@ -153,7 +153,8 @@ <h3>Fluid</h3>
 
     </div>
 
-    <div class="form-group" *ngIf="gasDensityForm.controls.gasType.value == 'OTHER'"
+    <div class="form-group"
+    *ngIf="gasDensityForm.controls.gasType.value != 'OTHER' && gasDensityForm.controls.gasType.value != 'AIR'"
       [ngClass]="{'indicate-different': isSpecificGravityDifferent(), 'invalid':gasDensityForm.controls.specificGravity.invalid}">
```

Comment:
Shouldn't have the check for AIR. We want to display this field if AIR is selected